docs/settings.md

```diff
@@ -82,6 +82,7 @@ Legend:
 | `bridge_bot_password`                | string                                                           |            | No   |        | Automatically generated password of the matterbridge bot user profile                                                                                                             |
 | `default_attachment_folder`          | string                                                           | `/Talk`    | No   |        | Specify default attachment folder location                                                                                                                                        |
 | `start_calls`                        | int                                                              | `0`        | Yes  | 🖌️    | Who can start a call, see [constants list](constants.md#start-call)                                                                                                               |
+| `max_call_duration`                  | int                                                              | `0`        | No   |        | Maximum duration of a call in seconds, 0 for unlimited                                                                                                                            |
```

Comment:
Should it be mentioned that federated rooms are not supported?

Reply:
They are on the host? But yes we can add it.